src/accordion/accordion-section.js

```diff
@@ -38,8 +35,9 @@ const component = {};
 component.require = {
     accordion: '^bdAccordion'
 };
+
 component.transclude = true;
-component.controllerAs = 'vm';
+component.controllerAs = 'section';
```

Comment:
I made some scope changes to the toggle/group directives so that they share scope (makes the watches work consistently). Apparently it caused collisions when the controllers were all named "vm."